app/src/main/java/com/asfoundation/wallet/ui/iab/PaymentMethodsView.kt

```diff
@@ -19,12 +19,12 @@ interface PaymentMethodsView {
   fun finish(bundle: Bundle)
   fun showLoading()
   fun hideLoading()
-  fun getCancelClick(): Observable<PaymentMethod>
+  fun getCancelClick(): Observable<PaymentMethod?>
```

Comment:
How can the PaymentMethod be nullable if Observable cannot emit null values? Same for the next two changes.